Cubical/Foundations/RelationalStructure.agda

```diff
@@ -45,60 +39,132 @@ InducedQuotientStr : (S : Type ℓ → Type ℓ') (ρ : StrRel S ℓ'')
   (A : TypeWithStr ℓ S) (R : Rel (typ A) (typ A) ℓ)
   → Type (ℓ-max ℓ' ℓ'')
 InducedQuotientStr S ρ A R =
-  Σ (S (typ A / R)) (ρ .rel (λ a b → [ a ] ≡ b) (A .snd))
-
--- A bisimulation R between a pair of structured types A, B /descends to the quotients/ when the induced
--- equivalence relations Rᴸ and Rᴿ on A and B induce structures on A/Rᴸ and B/Rᴿ and there is a path
--- between these structures over the equivalence A/Rᴸ ≃ B/Rᴿ
-record BisimDescends (S : Type ℓ → Type ℓ') (ρ : StrRel S ℓ'')
-  (A B : TypeWithStr ℓ S) (R : Bisimulation (typ A) (typ B) ℓ) : Type (ℓ-max ℓ' ℓ'')
+  Σ (S (typ A / R)) (ρ .rel (graphRel [_]) (A .snd))
+
+-- A structured equivalence relation R on a structured type A should induce a structure on A/R
+InducesQuotientStr : (S : Type ℓ → Type ℓ') (ρ : StrRel S ℓ'') → Type _
+InducesQuotientStr {ℓ = ℓ} S ρ =
+  (A : TypeWithStr ℓ S) (R : EquivPropRel (typ A) ℓ)
+  → ρ .rel (R .fst .fst) (A .snd) (A .snd)
+  → isContr (InducedQuotientStr S ρ A (R .fst .fst))
```

Comment:
Wouldn't this read nicer with https://github.com/agda/cubical/blob/master/Cubical/Data/Sigma/Base.agda#L44 ? You would have to unfold InducedQuotientStr though...